extensions/src/main/java/cm/aptoide/pt/extensions/StringExtensions.kt

```diff
@@ -23,6 +24,32 @@ fun String.isYoutubeURL(): Boolean {
   return matches(Regex(pattern))
 }
 
+fun String.extractVideoId(): String {
```

Comment:
This function could use Uri (from android) which has support functions to get query parameters and path segments directly